src/styles/index.scss

```
@@ -0,0 +1,9 @@
+@import "./utils/variables.scss";
+@import "./utils/functions.scss";
+@import "./utils/reset.scss";
```

I think order should be changed

```suggestion
@import "./utils/reset.scss";
@import "./utils/functions.scss";
@import "./utils/variables.scss";
```